packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -1,6 +1,5 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import Div100vh from 'react-div-100vh';
+import React, { ReactNode } from 'react';
+import { use100vh } from 'react-div-100vh';
```

Comment:
after updating `react-div-100vh`, some test cases failed So I had to change the logic based on the latest document